cli/buffcli/src/bin/buff/main.rs

```diff
@@ -0,0 +1,45 @@
+use quicli::prelude::*;
+use structopt::StructOpt;
+
+use bufflib::artifact;
+use bufflib::package_metadata;
+mod commands;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "buff", about = "Buff registry CLI")]
```

Comment:
is that really the Registry CLI? do we have any other types of CLI? does buff has more things other than registry?